src/js/popup.js

```diff
@@ -777,9 +777,20 @@ Logic.registerPanel(P_CONTAINER_INFO, {
       tr.classList.add("container-info-tab-row");
       tr.innerHTML = escaped`
         <td></td>
-        <td class="container-info-tab-title truncate-text" title="${tab.url}" >${tab.title}</td>`;
+        <td class="container-info-tab-title truncate-text" title="${tab.url}" >${tab.title}</td>
+        <td><img src="/img/container-close-tab.svg" style="width:50%;" id="close-tab" class="container-close-tab" /></td>`;
       tr.querySelector("td").appendChild(Utils.createFavIconElement(tab.favIconUrl));
 
+      document.getElementById("container-info-table").appendChild(fragment);
+
+      const closeTab = document.querySelector("#close-tab");
+      closeTab.setAttribute("id", tab.id);
+
+      Logic.addEnterHandler(closeTab, async function(e) {
```

Comment:
Can you move this logic inside line 759 rather than rely on CSS to prevent the click.

Most/All of the logic of this addon is within the JS and the current logic is already dictated by if it's `!tab.hiddenState`. By moving some of the logic into the CSS it will make it harder to maintain as developers will have to look in both places.

Also we should probably not even add the button / remove it if the row is a hidden tab. Making it `opacity: 0;` probably only makes it harder to click / it doesn't have a `tab.id` so does nothing perhaps?